frontend/src/profile-form-FE/ImgUpload.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from "react";
```

Comment:
Looks good to me

Reply:
front-end looks clean, can remove unused comments and recheck for following React naming conventions as said below too.